src/scanner.c

```diff
@@ -986,7 +996,7 @@ void GetSymbol ( void )
   case '^':         STATE(Symbol) = S_POW;           GET_NEXT_CHAR(); break;
 
   case '~':         STATE(Symbol) = S_TILDE;         GET_NEXT_CHAR(); break;
-  case '?':         STATE(Symbol) = S_HELP;          GetHelp(); break;
+  case '?':                                          GetHelp(); break;
```

Comment:
Why's this gone? I can't see any bug it could have been causing?

Reply:
It's not gone, I moved it into `GetHelp` for symmetry with the other `GetFOO` functions in the following lines.